mapping/enable/async_loader.py

```diff
@@ -2,7 +2,8 @@
 import asyncore
 import Queue
 import socket
-from threading import Thread
+from threading import Event, Thread
+from time import time as _time
```

Comment:
No - apparently fixing a bug in code that was elided (either that, or github isn't showing all changes).